package.json

```diff
@@ -22,7 +22,7 @@
         "clean": "rimraf dist",
         "prebuild": "npm run clean && npm run prettier:check && npm run intl:compile",
         "copy-messages": "cp -r src/messageFiles/compiled dist/messages",
-        "build": "tsc --project .",
+        "build": "tsc --project . && npm run intl:compile && npm run copy-messages",
```

Comment:
This is weird, the `prebuild` and `postbuild` scripts should still be ran under normal circumstances.
Does your PNPM setup in `prezly-dev` ignore lifecycle scripts?
If so, it makes sense to create a separate script for it, which would run `npm run prebuild && npm run build && npm run postbuild`, to keep everything unified.